Object Storage/using-object-storage-from-3rd-party-tools.md

```diff
@@ -54,25 +54,31 @@ CenturyLink Cloud now has a robust, geo-distributed Object Storage service capab
 2. Install CyberDuck and run the program. It looks like this when it starts up without any **Connections** (aka: Service Points or Servers) configured.
    ![cyberduck new install](../images/cyberduck-new-install.png)
 
-3. In the Control Portal, navigate to the Object Storage service and view the user record that you want to configure CyberDuck to use. Record the **Access Key ID** and the **Secret Access Key** for use in the tool.
+3. Download the CyberDuck profile [AWS S3 AWS2 Signature Version Profile for HTTPS](https://svn.cyberduck.io/trunk/profiles/S3%20AWS2%20Signature%20Version%20%28HTTPS%29.cyberduckprofile) to allow for AWS2 version authentication.
+
+4. In the Control Portal, navigate to the Object Storage service and view the user record that you want to configure CyberDuck to use. Record the **Access Key ID** and the **Secret Access Key** for use in the tool.
    ![Access Keys](../images/kb-object-storage-access-keys.png)
 
-4. Also you will need the **Service Point** (aka server) where your bucket exists (or will exist). In the Control Portal, on the Object Storage bucket listing page, either create a new bucket, or click on an existing bucket. When the **Bucket Info and Settings** page is displayed, copy/save the portion of the **bucket URL** that follows the bucket name. In this example, the bucket **Service Point** (server name) is: **canada.os.ctl.io**.
+5. Also you will need the **Service Point** (aka server) where your bucket exists (or will exist). In the Control Portal, on the Object Storage bucket listing page, either create a new bucket, or click on an existing bucket. When the **Bucket Info and Settings** page is displayed, copy/save the portion of the **bucket URL** that follows the bucket name. In this example, the bucket **Service Point** (server name) is: **canada.os.ctl.io**.
    ![object storage bucket info](../images/object-storage-bucket-info-and-settings.png)
```

Comment:
check the preview as this isn't wrapping properly.  best to add a full line between the text and the image